packages/core/src/awsService/appBuilder/serverlessLand/metadataManager.ts

```diff
@@ -32,6 +28,14 @@ export interface ProjectMetadata {
 export class MetadataManager {
     private static instance: MetadataManager
     private metadata: ProjectMetadata | undefined
+    private static readonly metadataPath = path.join(
+        path.resolve(__dirname, '../../../../../'),
```

Comment:
you're probably going to wait to integrate with our copyFiles scripts. This path probably exists in the extension debug mode but I don't think it exists in the actual build. Instead, you will need to copy this file to the resources file and pull from that

Reply:
Yes, I have made the changes in the path with the copyFiles scripts in the next PR and it is working correctly too.